src/test/regress/create_schedule

```diff
@@ -5,3 +5,4 @@ test: local_dist_join_load
 test: partitioned_indexes_create
 test: connectivity_checks
 test: sequences_create
+test: ddl_create
```

Comment:
as we discussed in the private chat, lets change the scope something smaller like `index_create` or such/